functions/controllers/comment/getComment.js

```diff
@@ -0,0 +1,49 @@
+const _ = require('lodash');
+const HttpStatus = require('http-status-codes');
+const { message, status } = require('../../util/constants');
+
+const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = HttpStatus;
```

Comment:
this is more neat and less of codes `const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = require('http-status-codes');`